.github/workflows/ci-testing.yml

```diff
@@ -33,6 +33,7 @@ jobs:
           - { os: 'ubuntu-20.04', python-version: 3.9, requires: 'latest', release: 'pre', topic: [ 'core' ] }
           - { os: 'ubuntu-20.04', python-version: 3.9, requires: 'latest', release: 'stable', topic: [ 'image' ] }
           - { os: 'ubuntu-20.04', python-version: 3.9, requires: 'latest', release: 'stable', topic: [ 'image','image_extras' ] }
+          - { os: 'ubuntu-20.04', python-version: 3.9, requires: 'latest', release: 'stable', topic: [ 'image','image_extras_effdet' ] }
```

Comment:
This seems to have no effect as it will lead to just image deps being installed and effdet is installed only for serving. I think it would be cleaner to not add a new job but just set the effdet version in `requirements/datatype_image_extras` to be the one from github since we don't expect users to install the `image_extras` extras anyway.

Reply:
I'm sorry, should have mentioned - I tried putting the version of effdet in `datatype_image_extras.txt` and doing `pip install -r requirements/datatype_image_extras.txt` fails miserably :/ Lots of version conflicts. Hence I thought to create a new file, and put effdet there.

Reply:
From the offline conversation with @ethanwharris - we decided to limit PyTorch version to <1.12 until `icevision` supports `effdet` 0.3.0.